python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -85,8 +85,8 @@ def __str__(self):
 
     def append(self, value):
         self.addProcess(value)
-        self.process_objects = sorted(
-            self.process_objects, key=lambda process: process.order)
+        if hasattr(self.process_objects, 'order'):
+            self.process_objects = sorted(self.process_objects, key=lambda process: process.order)
```

Comment:
Actually, I can give you an example right now :smile: :+1: 
```
j = Job()
j.postprocessors.append( Notifier() )
```

Now this is obviously another bug but I agree with @milliams that if this code was added, we could get some odd behaviour...